lib/presentation/widget/initialize_get_it/initialize_viewmodel.dart

```diff
@@ -70,23 +73,39 @@ class InitializeViewModel extends BaseViewModel {
     });
   }
 
-  ThunkAction<AppState> getCredentialAction() {
+  ThunkAction<AppState> _getCredentialAction() {
     return (Store<AppState> store) async {
       await _getCredentialInteractor.execute().then((result) => result.fold(
-          (left) => store.dispatch(getCredentialFailureAction(left)),
-          (right) => store.dispatch(getCredentialSuccessAction((right)))));
+        (left) => store.dispatch(_authenticationFailureAction()),
+        (right) => store.dispatch(_getCredentialSuccessAction((right)))));
     };
   }
 
-  ThunkAction<AppState> getCredentialSuccessAction(CredentialViewState success) {
+  ThunkAction<AppState> _getCredentialSuccessAction(CredentialViewState success) {
     return (Store<AppState> store) async {
       _dynamicUrlInterceptors.changeBaseUrl(success.baseUrl.origin);
       _retryInterceptors.setPermanentToken(success.token);
+      store.dispatch(_getAuthorizedUserAction());
```

Comment:
`InitializeViewModel ` is just a bridge to navigate between `LoginWidget` and `HomeWidget`, so I really want to keep it simple, less logic as possible, So I think you should get authorized user in `HomeViewModel`. WDYT?